Recipe for protein smoothie with powdered milk

Torres84f opened the pull request:
#### Summary  
Balance "Allow using powdered milk in a new protein smoothie recipe"

#### Purpose of change  
To add a way to use the powder milk in the protein smoothie recipe

#### Describe the solution  
Added a new recipe for the protein smoothie which uses powdered milk and clean water

#### Describe alternatives you've considered  
Adding powdered milk to the milk list, but I have no idea how to or what I'm touching, so this sounded better. 

#### Testing  
-Tested new recipe with the new ingredients and worked

#### Additional context  
Had to redo the whole PR as the initial change was bugged, so I added a new recipe for it, should I nest it?

Torres84f committed:
Update protein.json

Added powdered milk to the protein smoothie recipe so it can be crafted if you have it instead of having to rehydrate it then add it

SariusSkelrets commented:
If I'm reading this well, it means that the player could make a protein smoothie only made out of powder.

There should be a liquid somewhere, or else it won't be a drink. It'll just be a mass of dry powder.

SariusSkelrets commented:
By the way, #81236 will likely fix the issue you're mentionning by letting you craft the rehydrated milk and the smoothie in a single set of keypresses.

Torres84f committed:
Update protein.json

Ok huge blunder first attempt effectively could effectively make an smoothie without any liquids

added a 2nd recipe to use clean water + powdered milk

Torres84f commented:
> By the way, #81236 will likely fix the issue you're mentionning by letting you craft the rehydrated milk and the smoothie in a single set of keypresses.

I just added a 2nd protein smoothie recipe, don't know if it's cluttering the UI we will let the experienced devs decide but you were absolutely right, this happens because I'm a noob LoL

and yeah that thing you mentioned is much needed

thanks amigo

Torres84f committed:
Update data/json/recipes/food/protein.json

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Torres84f committed:
Update data/json/recipes/food/protein.json

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

SariusSkelrets commented:
Don't worry, everyone has to start somewhere.

The new recipe fills a gap as you mentionned, so I can see its use. Worst case, it gets removed once the recursive recipes are added, but this PR will allow people to avoid the inconvienience you mentionned until such recipes are added.

If you have questions, don't hesitate to ask. There are many, me included, who love answering them and help those who want to contribute but would like some advice.

Torres84f commented:
> Don't worry, everyone has to start somewhere.
> 
> The new recipe fills a gap as you mentionned, so I can see its use. Worst case, it gets removed once the recursive recipes are added, but this PR will allow people to avoid the inconvienience you mentionned until such recipes are added.
> 
> If you have questions, don't hesitate to ask. There are many, me included, who love answering them and help those who want to contribute but would like some advice.

Thanks, will do next time for sure, for some reason I didn't even pay attention to that I was just focused on it working with no errors and then BAM! you're mixing all the stuff in the blender but forgot any sort of liquid and I was creating a smoothie out of thin air lol
